infra/terraform/dev-org/dev-project-cleanup/locals.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  org_id          = "tbd"
+  billing_account = "tbd"
+  cleanup_folder  = "tbd"
```

Comment:
@morgante cleaner function requires all folders to clean to be under to a parent folder. However current dev hierarchy is 
- org
    - user1
    - user2

we can move them to a common folder or we can modify the cleaner function to accept an org.

Reply:
Let's move them to a common folder.

Reply:
Currently moved mine into the folder. Will announce and move other gradually.